command/initialize.go

```diff
@@ -71,7 +71,7 @@ func inputConfigLocation() string {
 	for {
```

Comment:
How would I use this feature to simplify the journalctl call in #69?

I'm also not 100% sure about the actual use-case of feature, is there something not possible with the current cli feature set and xargs? I don't find the argument that we don't have to use xargs very strong for implementing this inside gotify/cli.

I think if someone doesn't have access to xargs then they probably don't use gotify/cli and instead use some "raw" http call.

Reply:
> How would I use this feature to simplify the journalctl call in #69?

```sh
journalctl -f -u sshd | awk 'NR % 5 == 0 { print "\0"} { print }' | gotify push
```

> is there something not possible with the current cli feature set and xargs

makes sense, maybe I will just cherry pick the formatting bugs to a different PR and do that